web/core/components/issues/attachment/attachment-list-item.tsx

```diff
@@ -33,10 +33,16 @@ export const IssueAttachmentsListItem: FC<TIssueAttachmentsListItem> = observer(
   const { getUserDetails } = useMember();
   const {
     attachment: { getAttachmentById },
+    toggleAttachmentDeleteModal,
   } = useIssueDetail();
   // state
   const [isDeleteIssueAttachmentModalOpen, setIsDeleteIssueAttachmentModalOpen] = useState(false);
```

Comment:
if you are maintaining modal state in store, why are we having it here again?

Reply:
Resolved